cmd/python/prepare_accounts.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# bash script helper to create accounts (using ../cmd/python)
+# Parameters taken from  vars.sh
+set -x
+set -e
+. ../../vars.sh
+
+#CHANNEL_SEED=$1
+#AMOUNT_OF_SOURCE_ACCOUNTS=
+#AMOUNT_OF_DESTINATION_ACCOUNTS=
+#PASSPHRASE=$3
+#HORIZON=$4
+#FUNDER_SEED=$5
+echo $CHANNEL_SEED > channel_seed
+PASS=$PASSPHRASE
+
+pipenv sync; pipenv run python create.py --channel-seeds-file channel_seed --accounts $AMOUNT_OF_SOURCE_ACCOUNTS --passphrase "$PASS" --horizon $HORIZON --source-account $FUNDER_SEED --json-output True >../../accounts.json
+pipenv run python create.py --channel-seeds-file channel_seed --accounts $AMOUNT_OF_DESTINATION_ACCOUNTS --passphrase "$PASS" --horizon $HORIZON --source-account $FUNDER_SEED --json-output True >../../dest.json
```

Comment:
the `resources/create` and other binaries shouldn't be committed to this PR. they should be built by the `Makefile`. remove them

Reply:
As a concept I agree, these artifacts should be created as part of a CI flow, should be versioned, and saved in an artifactory.
But... it was not the scope of this task
I will add it as part of a different Jira epic of the CI

Reply:
i don't like adding binaries to a repo. cloning it after wards takes a mouch longer time because the binaries need to be downloaded. i'd rather we'd build the required apps at runtime instead of committing them

Reply:
In this case, building would take longer, as dependencies have to be downloaded.  Then we have the problem that we can't actually build it anyway, except on my (non-work) virtual linux machine.